src/components/BooksForm.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const chooseACategory = () => {
+  const options = categories.map (cat => (
+    <option key={cat} value={cat}>
```

Comment:
since the Form is a container, please relocate this file into the `src/containers/` directory